packages/config/config/devices/0x0330/zv9001k4-dim.json

```diff
@@ -0,0 +1,46 @@
+// Sunricher ZV9001K4-DIM
+// Z-Wave Dim Remote Control
+{
+	"manufacturer": "Sunricher",
+	"manufacturerId": "0x0330",
+	"label": "ZV9001K4-DIM",
+	"description": "Z-Wave Dim Remote Control",
+	"devices": [
+		{
+			"productType": "0x0003",
+			"productId": "0xa310"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Launch 1",
+			"maxNodes": 5
+		}
+	},
```

Comment:
These devices support Association Group Info CC and Z-Wave+, so associations can be auto-discovered. Unless the auto-discovered labels are worse than this, the entire block here is unnecessary.